<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Environmental Data</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>461</width>
     <height>356</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>
     QWidget {
         background-color: #ecf0f1; /* Set your background color here */
     }
     QLabel {
         font-size: 18px;
         font-weight: bold;
         padding: 5px;
     }
     QLabel#label_temperature {
         color: #2ecc71;
     }
     QLabel#label_temperature2 {
         color: #f39c12;
     }
     QLabel#label_humidity {
         color: #3498db;
     }
     QLabel#label_pressure {
         color: #e74c3c;
     }
     QLabel#label_temperature_value, QLabel#label_temperature2_value, QLabel#label_humidity_value, QLabel#label_pressure_value {
         color: #34495e;
         font-size: 24px;
         font-weight: bold;
     }
    </string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_temperature">
        <property name="text">
         <string>Air Temperature(DHT11):</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_temperature2">
        <property name="text">
         <string>Humidity(DHT11):</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_humidity">
        <property name="text">
         <string>Water Temperature(DS18B20):</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_pressure">
        <property name="text">
         <string>Pressure(BMP280):</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_temperature_value">
        <property name="text">
         <string>N/A</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_humidity_value">
        <property name="text">
         <string>N/A</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_temperature2_value">
        <property name="text">
         <string>N/A</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_pressure_value">
        <property name="text">
         <string>N/A</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>161</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_airtemp">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_humidity_2">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_watertemp_3">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pressure_4">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
